hrms/templates/generators/job_opening.html

```diff
@@ -187,7 +187,7 @@
 								{{ _("Closes On") if status == "Open" else _("Closed On") }}
 							</div>
 							<div class="font-weight-bold">
-								{{ frappe.format_date(closes_on if status == "Open" else closed_on, "d MMM, YYYY") }}
+								{{ frappe.utils.formatdate(closes_on if status == "Open" else closed_on, "d MMM, YYYY") }}
```

Comment:
Okay weird

So

`frappe.utils.format_date` also works fine

because internally, formatdate = format_date in the framework:

https://github.com/frappe/frappe/blob/07cedc581dba55384073e828a8f5c6c7e75d7562/frappe/utils/data.py#L228-L255

While frappe.format_date points to a different function altogether which seems to be buggy (framework):
https://github.com/frappe/frappe/blob/07cedc581dba55384073e828a8f5c6c7e75d7562/frappe/utils/safe_exec.py#L60